### ISSUE TITLE
Provide early outreach materials on new seasonality

### ISSUE THREAD

**Issue by kvantricht** [WorldCereal/worldcereal-classification#52]
- lowres screenshot of new global crop calendars
- paragraph describing what's new and ongoing
- request for feedback

This we can share as news item and share on socials.

**Comment by jdegerickx** [comment]
as discussed we expect a short text that can serve as a news item on the WorldCereal website indicating the activities done so far and some screenshots (also some zooms) of the new crop calendars.
Deadline for delivery = 20/06

**Comment by angui3** [comment]
I've written a brief text for the outreach and also some bullet points. Maybe the text fits better for a LinkedIn post and the highlights for a Twitter publication. Tell me if some change is needed. The screenshots I've taken are of Europe and India. The second is rather generic but I think that Europe's one is nice because it shows the inclusion of the pre-dormancy period for boreal winter wheat. 

[Outreach.zip](https://github.com/user-attachments/files/15915480/Outreach.zip)

**Comment by jdegerickx** [comment]
thanks for this!
Could you extend the explanation a bit so it becomes easy to understand for people not necessarily being aware of the Phase I approach? 
For instance, you talk about "Phase I points" as if everyone knows what they are but the majority of people reading this won't have a clue.
So please start with a short summary on the approach you are taking to generate these calendars in the first place.
Thanks!

**Comment by angui3** [comment]
How about this? I don't know if we should be more concrete about methodology. I've tried to keep it simple but maybe I should be more specific.

[Crop Calendars Outreach.docx](https://github.com/user-attachments/files/15917007/Crop.Calendars.Outreach.docx)

**Comment by jdegerickx** [comment]
Thanks for the update!
Reading through this, I'm not sure I fully understand what you did (so I guess others won't either :)).
I understand you previously trained a model using just climate data and existing crop calendars to be able to predict crop calendars in regions where we lack this information.
Now, you extended the training data and you mention the HLS NDVI data, but it is unclear to me what you used that data for. Did it become part of the model as one of the predictors or was it simply used to visually validate whether the results make sense?

**Comment by jdegerickx** [comment]
draft of news item shared via mail, feedback requested --> https://vitoresearch.sharepoint.com/:w:/r/sites/21717-ccn-world-cereal/Shared%20Documents/General/Communication/Website_Phase2/News/News_Phase2/20240635_CropCalendars-v1/Crop_calendars_news_item.docx?d=w5a75674905974ba4abe15099ed5ca169&csf=1&web=1&e=3UqK0S

**Comment by angui3** [comment]
I'm sorry for not giving feedback before, we had Midsummer's holidays early this week in València. I have added some minor changes and clarified a little the HLS methodology. We didn't used it into the training but to select the points we were feeding the model with. We used it to analyse the NDVI curves, see if they matched with the curves we would expect in those regions (as provided by Crop Monitor) and also derived the SOS of EOS as is similary done with LSP method (when we saw a rise in NDVI curve that correspond with planting and when it returned to the bare soil value we asigned the harvest date).
